### ISSUE TITLE
ui: ALTER TABLE ... SET LOCALITY shows as an undefined activity

### ISSUE THREAD

**Issue by otan** [cockroachdb/cockroach#65692]
On the Admin UI, SET LOCALITY events show up as undefined.

Repro steps:

* Perform an ALTER TABLE...LOCALITY REGIONAL BY ROW
* Go to DB Console and look at the schema change logs. You'll see:
![image](https://user-images.githubusercontent.com/3646147/119579123-22061200-be01-11eb-9003-06a0a30df8d9.png)

**Comment by otan** [comment]
hmm, the `ID undefined` is actually the mutationID, which is not always set.

**Comment by otan** [comment]
unfortunately https://github.com/cockroachdb/cockroach/pull/63099/files only worked for ALTER DATABASE events

**Comment by arulajmani** [comment]
What's the behaviour we want here exactly? Should we be showing the table's ID instead of the mutation ID? Is there any value in showing the mutation ID in the event log anyway?

**Comment by otan** [comment]
i think it should be a different event on the event log, be good to get PM feedback

**Comment by arulajmani** [comment]
Makes sense, cc @awoods187 -- thoughts on how this event log should look?
